playbooks/roles/memcached/defaults/main.yml

```diff
@@ -0,0 +1,4 @@
+---
+
+COMMON_MEMCACHE_LISTEN_PORT: 11211
+COMMON_MEMCACHE_BIND_IP: '127.0.0.1'
```

Comment:
@melvinsoft unless these vars are part of the `common` or `common_vars` role, I think we should drop the `COMMON_` prefix. 

Also, we should really push this change upstream. It's such a pain that the edX playbooks don't have a template for memcached.conf that allows it to bind to something other than localhost.